com.ibm.wala.core/src/testJava11/privateInterfaceMethods/testArrayReturn/returnArray.java

```diff
@@ -0,0 +1,12 @@
+package testArrayReturn;
+
+public class returnArray {
```

Comment:
We are trying to test private interface methods, but this is a class, not an interface.  Maybe try a test case exactly like what was in [the blog post](https://jakewharton.com/androids-java-9-10-11-and-12-support/)?
```java
interface Java9PrivateInterface {
  static String hey() {
    return getHey();
  }

  private static String getHey() {
    return "hey";
  }
}
```
Notice that `Java9PrivateInterface` is an `interface`, not a `class`.

Also, the capitalization in the class and method names here is not the standard style for Java.  Class names should start with a capital letter, while method names should be camel case.  See the naming guide [here](https://google.github.io/styleguide/javaguide.html#s5-naming).